manala.skeleton/CHANGELOG.md

```diff
@@ -5,6 +5,8 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/)
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ## [Unreleased]
+### Removed
+- Rtail support (deprecated)
```

Comment:
I'd say `changed` for the deprecation time, and add a `removed` when truly remove it (next major?)

Reply:
Oh dear, i already had an oral 👍 ...

Reply:
Anyway, you're absolutely right !

Reply:
See: https://github.com/manala/ansible-roles/commit/769faf7df7525382f1d06d112ad18c14540adf51

![](https://media.giphy.com/media/49HINwAf1JOuI/giphy.gif)